server/controllers/auth.js

```diff
@@ -0,0 +1,27 @@
+import authService from '../services/auth';
+
+function loginUser(req, res) {
+  authService.loginUser(req.body.email, req.body)
+    .then((token) => {
+      if (token.name === 'Error') {
+        return res.status(404).send('No such user exists');
+      }
+      return res.status(200).json(token);
```

Comment:
Yeah that seems reasonable to me. @kwm4385 @paradime what do you guys need from this?